ui/src/components/common/Header.js

```diff
@@ -14,43 +14,28 @@ import { UserIcon } from '@patternfly/react-icons';
 import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
 import { css } from '@patternfly/react-styles';
 import './Header.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toggleHeaderDropdown } from '../../actions/actions-ui';
 
 class Header extends React.Component {
-  constructor (props) {
-    super(props);
+  onTitleClick = () => {};
 
-    this.state = {
-      isUserDropdownOpen: false
-    };
-  }
-
-  onTitleClick = () => {
-    console.log('onTitleClick()');
-    // const { history } = this.props;
-    // history.push('/');
+  onUserDropdownToggle = () => {
+    const isUserDropdownOpen = this.props.isUserDropdownOpen;
+    this.props.toggleHeaderDropdown(isUserDropdownOpen);
```

Comment:
@damienomurchu I did the same when I initially set up the `isappsrequestfailed` where I passed the value through the component, action and reducer. As you are reversing it in the reducer you don't need to pass the value down.

Reply:
good point @laurafitzgerald :+1: